src/ansys/mapdl/core/pool.py

```diff
@@ -614,7 +614,7 @@ def run():
                     try:
                         self._exiting_i += 1
                         instance.exit()
-                    except Exception as e:
+                    except Exception:
                         LOG.error("Failed to close instance", exc_info=True)
```

Comment:
Since we are already catching the exception.

```suggestion
                        LOG.error(f"Failed to close instance due to:\n{e}", exc_info=True)
```